tests/com/tony/server/AuthenticatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.tony.server;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AuthenticatorTest {
+    @Test
+    public void testAddToAuthenticationList() throws Exception {
+        Authenticator authenticator = new Authenticator();
```

Comment:
On second thought, you should just pull out `new Request("GET", "/blah")` into a `Before`, or even a `static final` variable, as long as it's state doesn't get modified by the tests.